package/MDAnalysis/coordinates/base.py

```diff
@@ -1741,7 +1753,63 @@ def get_aux_descriptions(self, auxnames=None):
             auxnames = self.aux_list
         descriptions = [self._auxs[aux].get_description() for aux in auxnames]
         return descriptions
-
+    
+    @property
+    def transformations(self):
+        """ Returns the list of transformations"""
+        return self._transformations
+    
+    @transformations.setter
+    def transformations(self, transform_list):
+        if not self._transformations:
+            self._transformations = transform_list
```

Comment:
Shouldn't it be `add_transformations`?

Reply:
I found it easier that way, because I needed to override the ```add_transformation``` method in the ```MemoryReader``` to deal with the "overtransforming" issue